lte/cloud/go/services/lte/obsidian/handlers/handlers.go

```diff
@@ -300,21 +304,41 @@ func listEnodebs(c echo.Context) error {
 	if nerr != nil {
 		return nerr
 	}
+	var pageSize uint64 = 0
+	var err error
 
-	ents, _, err := configurator.LoadAllEntitiesOfType(
+	if pageSizeParam := c.QueryParam(ParamPageSize); pageSizeParam != "" {
+		pageSize, err = strconv.ParseUint(pageSizeParam, 10, 32)
+		if err != nil {
+			err := fmt.Errorf("invalid page size parameter: %s", err)
+			return obsidian.HttpError(err, http.StatusBadRequest)
+		}
+	}
+
+	ents, nextPageToken, err := configurator.LoadAllEntitiesOfType(
 		nid, lte.CellularEnodebEntityType,
-		configurator.EntityLoadCriteria{LoadMetadata: true, LoadConfig: true, LoadAssocsToThis: true},
+		configurator.EntityLoadCriteria{
+			LoadMetadata:     true,
+			LoadConfig:       true,
+			LoadAssocsToThis: true,
+			PageSize:         uint32(pageSize),
+			PageToken:        c.QueryParam(ParamPageToken)},
 		serdes.Entity,
 	)
 	if err != nil {
 		return obsidian.HttpError(err, http.StatusInternalServerError)
 	}
 
-	ret := make(map[string]*lte_models.Enodeb, len(ents))
+	enodebs := make(map[string]*lte_models.Enodeb, len(ents))
 	for _, ent := range ents {
-		ret[ent.Key] = (&lte_models.Enodeb{}).FromBackendModels(ent)
+		enodebs[ent.Key] = (&lte_models.Enodeb{}).FromBackendModels(ent)
 	}
-	return c.JSON(http.StatusOK, ret)
+	paginatedEnodebs := &lte_models.PaginatedEnodebs{
+		Enodebs:       enodebs,
+		NextPageToken: lte_models.NextPageToken(nextPageToken),
+		TotalCount:    int64(len(enodebs)),
```

Comment:
This isn't quite right. TotalCount is meant to be an estimate of how many of the object exist "in total", not just how many objects were returned on this page. Fix this and then add a regression test to display the proper behavior